### DISCUSSION TITLE
Moduli ingressi Zigbee

### DISCUSSION THREAD

**Discussion by ADondoni** [indomus/forum#2098] [Hardware & Firmware]
Buongiorno

Sto cercando da un pò di tempo dei "moduli" x degli ingressi zigbee ( da 2 o 3), perche vorrei rendere smart i miei pulsanti/interruttori di casa, ma non trovo praticmente nulla, ho trovato solo lo shelli I3 che però è WI-FI, qualcuno mi sà indicare qulacosa?

**Comment by KC-inDomus** [comment]
sai che non ho capito cosa cerchi

**Reply by Scialla** [reply]
cerca degli "shelly" che funzionino in zigbee e non in wifi....

Devi cercare su aliexpress, qualcosa c'è ma, devi vedere anche che sia supportato dal tuo coordinator (ZHA o z2m?) guardando su blackadder
https://zigbee.blakadder.com/

Ad esempio il sonoff zigbee mini che è pure in offerta black friday

**Reply by snakuzzo** [reply]
sì ma quella lista non è mai completa/aggiornata

**Reply by Scialla** [reply]
vero ma, l'alternativa è andare a leggersi la changelog di z2m (ad esempio) per vedere i nuovi dispositivi supportati..

**Reply by ADondoni** [reply]
Grazie a tutti per le risposte, sostanzialmente cerco qualcosa di identico allo Shelly I3, ma che comunichi in zigbee (non WI-FI)
